pages/manage/productions/create.vue

```diff
@@ -1,15 +1,110 @@
 <template>
   <div class="wrapper">
-    <div class="container-fluid">
+    <section class="container-fluid">
       <h1 class="title">
-        {{ 'create' }}
+        {{ $t('productions.create') }}
       </h1>
-    </div>
+
+      <p>
+        <strong>{{ $t('productions.suggested_events') }}</strong>
+        {{ similarityScore }}%
+      </p>
+
+      <section class="events-container">
+        <event
+          v-for="event in suggestedEvents"
+          :id="parseEventId(event['@id'])"
+          :key="event['@id']"
+          :type="getEventType(event.terms)"
+          :title="event.name[locale]"
+          :start-date="event.startDate"
+          :end-date="event.endDate"
+          :image-url="event.image"
+          :description="event.description[locale]"
+        />
```

Comment:
Out of curiosity, can you explain why you an `event` component with `v-for` here and pass it only the props it needs, while in the ` pages/manage/productions/index.vue` you use an additional `events` (plural) component and pass it the whole event(s) data?

I have no preference for one or the other, but ideally, we'd pick one approach and stick with it for consistency I believe.

Reply:
The `events` component is a collection of multiple events, a list. In the create page there are always only 2 events. Either way, I've changed both `event` components to take id, name, ... as separate props now.